MapboxAndroidDemo/src/main/res/layout/activity_localization_plugin.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:mapbox="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <com.mapbox.mapboxsdk.maps.MapView
+        android:id="@+id/mapView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:mapbox_cameraTargetLat="40.745565"
+        app:mapbox_cameraTargetLng="-73.982058"
+        app:mapbox_cameraZoom="11.316102"
+        app:mapbox_styleUrl="@string/mapbox_style_mapbox_streets">
+
+    </com.mapbox.mapboxsdk.maps.MapView>
+
+    <android.support.constraint.ConstraintLayout
```

Comment:
Why not just make the entire layout a constraintLayout? No need to have a FrameLayout as the parent.

Reply:
That's what I originally had, but then decided to have the instructional cardview and buttons to be on top of the map in a frame layout, rather than the top of the map ending at the bottom of the buttons in a constraint layout...